core/src/main/kotlin/ch/ergon/dope/resolvable/clause/LimitClause.kt

```diff
@@ -0,0 +1,18 @@
+package ch.ergon.dope.resolvable.clause
+
+import ch.ergon.dope.resolvable.expression.TypeExpression
+import ch.ergon.dope.resolvable.formatToQueryString
+import ch.ergon.dope.validtype.NumberType
+
+private const val LIMIT = "LIMIT"
+
+class LimitClause(private val numberExpression: TypeExpression<NumberType>, private val parentClause: IOrderByClause) : ILimitClause {
```

Comment:
If you have a _DeleleLimitClause_ then I would name this _SelectLimitClause_, the same with the other classes and interfaces

Reply:
done